scripts/jenkins-support.sh

```diff
@@ -24,13 +25,14 @@ if ! openstack image show -c name --format value $image_name; then
 fi
 
 # Run rally test
-rally task start rally-test.json
+rally task start $task
```

Comment:
You need to quote the `$` if you want to use the variable `task` from `setenv`. Unquoted it will be expanded by the shell immediately.

Reply:
fixed